front/app/components/EventAttendanceButton/index.tsx

```diff
@@ -192,7 +206,11 @@ const EventAttendanceButton = ({ event }: EventAttendanceButtonProps) => {
           icon={getButtonIcon()}
           iconSize="20px"
           bgColor={
-            userIsAttending ? colors.success : theme.colors.tenantPrimary
+            maxAttendeesReached && !userIsAttending
+              ? colors.grey800 // Grey background when no spots left
```

Comment:
Not good enough contrast, but not sure what to use that is better. Asking [here](https://go-vocal.slack.com/archives/CQAGMHKT3/p1750964382970839).